components/molecules/FriendAvatar/FriendAvatar.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
```

Comment:
I don't think we should create a new FriendAvatar component, we already have a `Friend` and `FriendGroup` component within lava-components. Those should be the singular avatar components that we lean into.